src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -68,58 +69,83 @@ const coreClinicalSchemaNamesSet = new Set<CoreClinicalSchemaName>(
   Object.keys(schemaNameToCoreCompletenessStat) as CoreClinicalSchemaName[],
 );
 
-// This is the main core stat caclulation function.
+const getEmptyCoreStats = (): CompletionStats =>
+  cloneDeep({
```

Comment:
unclear why cloning here. Won't this be a new object every time the function is called because lack of closure?

Reply:
The original function:

`const getEmptyCoreStats = (): CoreCompletionFields => {
  return cloneDeep({
    donor: 0,
    specimens: 0,
    primaryDiagnosis: 0,
    followUps: 0,
    treatments: 0,
  });
};`

Here I'm just trying not to break anything

Reply:
looks like cloneDeep is still here?

change here is shorthand hand return vs explicit return - 
```
const getEmptyCoreStats = (): CoreCompletionFields => {
  return cloneDeep({
    donor: 0,
    specimens: 0,
    primaryDiagnosis: 0,
    followUps: 0,
    treatments: 0,
  });
};
```
vs
```
const getEmptyCoreStats = (): CoreCompletionFields => 
  cloneDeep({
    donor: 0,
    specimens: 0,
    primaryDiagnosis: 0,
    followUps: 0,
    treatments: 0,
  });
};
```

maybe commit didn't push?

Reply:
these are comments on stale code.
latest commit has changes.

Reply:
Thanks!
https://github.com/icgc-argo/argo-clinical/pull/954/commits/f32c7c8fb699a4ba0b52bb494d6e4b247bc9582e#diff-298db3eb8224598049e8023b969f0b4b4423591607896635705aefbbc5e13c9a